server/code/setup/cpdb

```diff
@@ -34,6 +35,7 @@ else:
 
 JBOSS_CLASSPATH = "/var/lib/jbossas/server/production/deploy/candlepin.war/WEB-INF/classes/"
 TOMCAT_CLASSPATH = "/var/lib/" + TOMCAT + "/webapps/candlepin/WEB-INF/classes/"
+JAR_PATH = "/var/tmp/candlepin/WEB-INF/lib/"
```

Comment:
The path on an installed candlepin (as seen via `rpm -ql candlepin` is `/var/lib/tomcat/webapps/candlepin/WEB-INF/lib`), please build the path similar to how the `TOMCAT_CLASSPATH` is built.

Reply:
Done.